cves/kernel/CVE-2016-3135.yml

```diff
@@ -270,8 +271,11 @@ sandbox:
     Answer should be true or false
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: |
```

Comment:
Can this be explained with more detail if possible of how heap corrections are used to gain privileges?

Reply:
Added a better explanation.